index.html

```diff
@@ -86,7 +86,7 @@ <h2 id="features">Features</h2>
   <li>Multiple container runtimes. Marathon has first-class support for both <a href="https://mesosphere.github.io/marathon/docs/application-basics.html">Mesos containers</a> (using cgroups) and <a href="https://mesosphere.github.io/marathon/docs/native-docker.html">Docker</a>.</li>
   <li><a href="https://mesosphere.github.io/marathon/docs/persistent-volumes.html">Stateful apps</a>. Marathon can bind persistent storage volumes to your application. You can run databases like MySQL and Postgres, and have storage accounted for by Mesos.</li>
   <li><a href="https://mesosphere.github.io/marathon/docs/marathon-ui.html">Beautiful and powerful UI</a>.</li>
-  <li><a href="https://mesosphere.github.io/marathon/docs/constraints.html">Constraints</a>. e.g. Only one instance of an application per rack, node, etc.</li>
+  <li><a href="https://mesosphere.github.io/marathon/docs/constraints.html">Constraints</a>. E.g., only one instance of an application per rack, node, etc.</li>
```

Comment:
is there really supposed to be comma after e.g.? is the E really supposed to be uppercase?